src/components/Home.js

```diff
@@ -18,25 +18,33 @@ const Home = () => {
     }
 
     const handleClick = () => {
-        url += '&ingr=' + ingredients; // 10-20
+        url += '&ingr=' + ingredients;
+        if (query !== '') {
+            url += '&q=' + query;
+        }
         if (diet !== 'none') {
-            url += '&health=' + diet; // vegetarian
+            url += '&health=' + diet;
         }
         if (cuisine !== 'none') {
-            url += '&cuisineType=' + cuisine; // Asian
+            url += '&cuisineType=' + cuisine;
         }
-        url += '&mealType=' + mealType; // Dinner
-        url += '&time=' + time; // 10-20
+        url += '&mealType=' + mealType;
```

Comment:
let url = `https://api.edamam.com/api/recipes/v2?type=public& ${app_id} ID& ${app_key} KEY ;
    url += '&app_id=' + ID;
    url += '&app_key=' + KEY;

Reply:
I have resolved this in my latest commit.